src/zkscript/fields/fq4.py

```diff
@@ -440,13 +475,24 @@ def frobenius_even(
     ) -> Script:
         """Frobenius for even powers n = 2k in F_q^4.
 
-        Input parameters:
-            - Stack: q .. X
-            - Altstack: []
-        Output:
-            - X**(q**2)
-        Assumption on data:
-            - X is passed as as a couple of elements of Fq2 (see Fq2.py). Namely, X = a b
+        Stack input:
+            - stack:    [q, ..., x := (x0, x1, x2, x3)], `x` is a couple of elements of F_q^2
+            - altstack: []
+
+        Stack output:
+            - stack:    [q, ..., x^q^2]
```

Comment:
double exponentiation need some parenthesis (I think this happened in a few more parts of the code)

Reply:
I guess this should be (x^q)^n. @BarbacoviF can you confirm?